python/friesian/example/wnd/recsys2021/README.md

```diff
@@ -85,25 +81,22 @@ python wnd_preprocess_recsys.py \
     --executor_memory 150g \
     --num_nodes 8 \
     --train_files 0-269 \
-    --input_train_folder /path/to/the/folder/of/train/parquet_files \
-    --input_test_folder /path/to/the/folder/of/train/parquet_files \
-    --output_folder /path/to/the/folder/to/save/preprocessed/parquet_files \
+    --data_dir /path/to/the/folder/of/parquet_files \
     --cross_sizes 600
 ```
 
 __Options:__
 * `cluster_mode`: The cluster mode to run the data preprocessing, one of local, yarn or standalone. Default to be local.
 * `master`: The master URL, only used when cluster_mode is standalone.
+* `backend`: The backend of Orca Estimator, either ray or spark. Default to be ray.
 * `executor_cores`: The number of cores to use on each node. Default to be 44.
 * `executor_memory`: The amount of memory to allocate on each node. Default to be 130g.
 * `num_nodes`: The number of nodes to use in the cluster. Default to be 8.
 * `driver_cores`: The number of cores to use for the driver. Default to be 4.
 * `driver_memory`: The amount of memory to allocate for the driver. Default to be 36g.
 * `train_files`: The part range for train data preprocessing, such as 0-269 for the full training dataset,
  0-1 for the first two parts, etc. Default to be 0-269.
-* `input_train_folder`: The path to the folder of train parquet files, either a local path or an HDFS path.
-* `input_test_folder`: The path to the folder of test parquet files, either a local path or an HDFS path.
-* `output_folder`: The path to save the preprocessed data to parquet files. HDFS path is recommended.
+* `data_dir`: The path to the folder of parquet files,  either a local path or an HDFS path, where train/test/preprocessed parquets are stored under `train/`, `test/`, `preprocessed` respectively.
```

Comment:
The path to the folder of parquet files,  either a local path or an HDFS path. Train and test parquet files should be stored under `train/`, `test/` and processed parquet files would be stored under `preprocessed/`.